test/tools/IsolTestOptions.cpp

```diff
@@ -51,10 +56,10 @@ IsolTestOptions::IsolTestOptions(std::string* _editor):
 	CommonOptions(description)
 {
 	options.add_options()
+		("editor", po::value<std::string>(_editor)->default_value(editorPath()), "Path to editor for opening test files.")
 		("help", po::bool_switch(&showHelp), "Show this help screen.")
-		("no-color", po::bool_switch(&noColor), "don't use colors")
-		("editor", po::value<std::string>(_editor)->default_value(editorPath()), "editor for opening test files");
-
+		("no-color", po::bool_switch(&noColor), "Don't use colors.")
+		("test,t", po::value<std::string>(&test)->default_value(std::string{}), "Filters which test units to include.");
```

Comment:
This expresses a long and a short option: `"test,t"` -> `--test | -t`

Reply:
While we're add it you could also add ``-e`` for ``--editor`` and especially ``-h`` for ``--help``... but optional for this PR, but it'd be fine to just put it in, if you're still changing stuff anyways.